Overview.bs

```diff
@@ -1956,30 +1960,127 @@ functionality/data from all of its patch files even if those were included in th
 directly encoded into the IFT format from font authoring source files.
 
 
-Encoder Considerations {#encoder-considerations}
+Encoding Considerations {#encoding-considerations}
 ------------------------------------------------
 
 <em>This section is not normative.</em>
 
 The details of the encoding process may differ by encoder and are beyond the scope of this document. However, this section provides
-guidance that encoder implementations may want to consider.
+guidance that encoder implementations may want to consider, and that can be important to reproducing the appearance and behavior of
+an existing font file when producing an [=incremental font|incremental=] version of that font.
+
+<b>About [[#brotli]] and [[#per-table-brotli]] patches</b>
```

Comment:
Would be good to link back to the section on invalidation (4.1) somewhere in this sub-section.

Reply:
You could also link to the specific invalidation type definitions whenever you reference them (eg. [=Partial Invalidation|Partially Invalidating=])